lsp/package.json

```diff
@@ -1,64 +1,35 @@
 {
-    "name": "awsdocuments-language-server",
-    "description": "Language Server for AWS Filetypes",
-    "author": "AWS",
-    "license": "MIT",
-    "version": "0.1.0",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/aws/awsdocuments-language-server"
-    },
-    "publisher": "aws",
-    "categories": [],
-    "keywords": [],
-    "engines": {
-        "vscode": "^1.74.0"
-    },
-    "bin": {
-        "awsdocuments-language-server": "./out/src/server/server.js"
-    },
-    "main": "./out/src/server/server.js",
+    "name": "monorepo-aws-languages",
+    "version": "1.0.0",
+    "description": "A monorepo for AWS Language Servers (@christou-lsp-test)",
+    "private": true,
+    "workspaces": [
+        "app/*",
+        "client/**",
+        "core/*",
+        "server/*"
+    ],
     "scripts": {
         "prepare": "cd .. && husky install lsp/.husky",
-        "vscode:prepublish": "npm run compile",
-        "compile": "tsc -b",
-        "package-x64": "pkg --targets node18-linux-x64,node18-win-x64,node18-macos-x64 --out-path bin --compress GZip .",
-        "package-arm": "pkg --targets node18-linux-arm64,node18-win-arm64,node18-macos-arm64 --out-path bin --compress GZip .",
-        "watch": "tsc -b -w",
         "lint": "eslint src --ext .ts,.tsx && prettier --check .",
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
-        "test": "mocha out/test/unit/**/*.js",
-        "testInt": "mocha out/test/integration/**/*.js"
+        "compile": "tsc --build --verbose",
+        "watch": "tsc --build --watch"
     },
     "dependencies": {
-        "request-light": "^0.6.0",
-        "vscode-languageserver": "^7.0.0",
-        "vscode-languageserver-textdocument": "^1.0.4",
-        "vscode-uri": "^3.0.7",
-        "yaml-language-server": "^1.13.0"
+        "typescript": "^5.0.4"
     },
     "devDependencies": {
-        "@types/chai": "^4.3.4",
-        "@types/mocha": "^9.1.0",
-        "@types/node": "^16.11.7",
-        "@types/sinon": "^10.0.13",
-        "@types/vscode": "^1.74.0",
-        "@typescript-eslint/eslint-plugin": "^5.42.0",
-        "@typescript-eslint/parser": "^5.42.0",
-        "chai": "^4.3.7",
-        "create-temp-directory": "^2.2.0",
-        "eslint": "^8.26.0",
+        "@types/node": "^20.2.5",
+        "@typescript-eslint/eslint-plugin": "^5.59.9",
+        "@typescript-eslint/parser": "^5.59.9",
+        "eslint": "^8.42.0",
         "eslint-plugin-unused-imports": "^2.0.0",
         "husky": "^8.0.3",
-        "mocha": "^10.2.0",
```

Comment:
Do the tests still run with some of these dependencies being removed?

Reply:
The tests will not run currently. I will have a follow up change that moves the "core" code around, wires up the existing tests, and sets the CI to run the tests.